.github/workflows/build.yml

```diff
@@ -148,6 +148,12 @@ jobs:
       with:
         fetch-depth: 0
 
+    - name: "Ensure that all tags are fetched"
+      # Works around an issue where not the latest tag is not fetched when the
+      # workflow is triggered by a tag push event.
+      # Possibly related: actions/checkout#290
+      run: git fetch origin --force --tags
```

Comment:
This seems to work: https://github.com/Holzhaus/mixxx/runs/2365251296?check_suite_focus=true#step:28:25